tensorflow/python/keras/layers/wrappers.py

```diff
@@ -441,10 +441,15 @@ def set_weights(self, weights):
 
   @tf_utils.shape_type_conversion
   def compute_output_shape(self, input_shape):
-    output_shape = tuple(self.forward_layer.compute_output_shape(
-        input_shape).as_list())
+    forward_layer_output_shape \
```

Comment:
can you avoid using `\` to designate line breaks? This is tensorflow python style (just break after the parenthesis)

Reply:
`\` has been removed and add the break after the parenthesis.